File: 상현 알고리즘/약수의 개수와 덧셈.py
def solution(left, right):
    result = 0 
    count = 0
    for num in range(left,right+1):
        count = 0
        for i in range(1,num+1):
            if num % i ==0:
                count +=1
        if count % 2 ==0:
            result +=num
        else :
            result -= num
    return result

File: thyoondev/약수의 개수와 덧셈.js
const countMea = (num) => {
  let count = 0;
  for (let i = 1; i <= num + 1; i++) {
    if (num % i === 0) {
      count++;
    }
  }
  return count;
};

function solution(left, right) {
  var answer = 0;
  for (let i = left; i <= right; i++) {
    countMea(i) % 2 === 0 ? (answer += i) : (answer -= i);
  }
  return answer;
}
